src/interactions/dragInteraction.ts

```diff
@@ -44,6 +44,10 @@ export module Interaction {
     }
 
     private _startDrag(p: Point, e: UIEvent) {
+      var button = (<MouseEvent> e).button;
+      if (button != null && button !== 0) {
+        return;
+      }
```

Comment:
Maybe; Checking for the button property is a sufficient proxy though, since that's what we're actually interested in.